### PR TITLE
feat(e2e-connector-test): add fuel wallet tests

### PR DESCRIPTION
**Pull request by nelitow**
This PR creates tests for the Fuel Wallet connector in the react-app.
The tests are for transferring, incrementing and connection stability.

### CODE REVIEW

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
use this folder structure to separate logic of connectors from general logic

```
react-app
  connectors
    FuelWalletConnector
      FuelWalletConnector.test.ts
      setup.ts
      // .... any other file with specific logic from this connector
  utils
    // generic util files
```

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
in the setup.ts file

the part 1 you can keep in this file as its generic test commands
the part 2 it's better to execute directly inside of each connector test, as it is specific for each connector and will change

image

File: e2e-tests/react-app/FuelConnector.test.ts [hunk 1]
```diff
@@ -2,30 +2,91 @@ import { downloadFuel } from '@fuels/playwright-utils';
...
 import { test } from '@fuels/playwright-utils';
 import { expect } from '@playwright/test';
-import type { WalletUnlocked } from 'fuels';
-import { bn, toBech32 } from 'fuels';
-import { testSetup } from '../utils/index.js';
+import { type WalletUnlocked, bn } from 'fuels';
+import { testSetup, transferMaxBalance } from '../utils/index.js';
 import { connect } from './utils';
 
 const fuelPathToExtension = await downloadFuel('0.21.0');
```
Comment (LuizAsFight) [inline e2e-tests/react-app/FuelConnector.test.ts:9]: download the latest version always, 0.27.0 is being released right now you can use it

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: e2e-tests/react-app/FuelConnector.test.ts (hunk 1 above)
Comment (nelitow) [reply e2e-tests/react-app/FuelConnector.test.ts]: Done, but maybe a `latest` tag could be useful in the `downloadFuel` helper.

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**